config/babelrc.js

```diff
@@ -28,6 +29,7 @@ export default (env, options) => ({
     plugins: [
         // Make optional the explicit import of React in JSX files
         pluginReactRequire,
+        pluginTransformRuntime,
```

Comment:
Can we test it again ? Maybe it was for another reason ? And in any case, we should at least add it only for the test environment.